README.md

```diff
@@ -94,7 +94,7 @@ Please note: the test tables built in the CI pipeline will exist for ~24 hours.
 
 ### Connecting with other wizards
 
-We use Discord to connect with our community. Head over to spellbook channel on [Dune's Discord](https://discord.gg/dunecom) for questions or to ask for help with a particular PR. We encourage you to learn by doing, and leverage our vibrant community to help you get going.
+We use Discord to connect with our community. Head over to spellbook channel on [Dune's Discord](https://discord.com/invite/ErrzwBz) for questions or to ask for help with a particular PR. We encourage you to learn by doing, and leverage our vibrant community to help you get going.
```

Comment:
@operagxsasha thank you for raising this. can you sign the CLA attached to PR, in order to proceed?

@0xBoxer can you help validate what the link should be here, to join the discord?

Reply:
that's the same invite we use on our website, works for me!